src/main/java/de/tum/in/www1/artemis/service/user/UserService.java

```diff
@@ -1,5 +1,7 @@
 package de.tum.in.www1.artemis.service.user;
 
+import static de.tum.in.www1.artemis.config.Constants.*;
+import static de.tum.in.www1.artemis.config.Constants.PASSWORD_MAX_LENGTH;
```

Comment:
```suggestion
import static de.tum.in.www1.artemis.config.Constants.*;
```

Reply:
Weird combination of Auto-forrmatting and spotless probably